### ISSUE TITLE
Italicize titles 

### ISSUE THREAD

**Issue by oliviagallucci** [ossf/secure-sw-dev-fundamentals#73]
There are a lot of "2021 CWE Tip 25" and "2019 CWE Top 25". It gets hard to follow when they are not italicized because the sentences are so long. 

All of the CWE and OWASP lists should be in italics IMO.

**Comment by david-a-wheeler** [comment]
That would make some sense. That change would happen many times, so we probably need to do a find & replace. I'll rope in Flavia before we start doing that.
